.gitignore

```diff
@@ -59,3 +59,4 @@ typings/
 
 .netlify
 dist/
+live
```

Comment:
The .gitignore file only works for files and directories that are not already tracked by Git. Adding an entry to .gitignore afterwards has no effect. Consequently, your PR still contains the `Live` folder. You can just delete that folder in VSCode from your week2 branch, commit and push to update the PR.